HACKING.md

```diff
@@ -108,8 +112,46 @@ otherwise an official image can be run with
 
     $ make run
 
+Log in to the FAF container
+
+    $ make sh
+
+Set up the database in `/etc/faf/faf.conf`, under `[Storage]`. Use the database container's IP address found above as `DBHost`, set `DBPort` to `5432`, `DBName` to `faf`, `DBUser` to `faf` and `DBPasswd` to `scrt`.
```

Comment:
Why not provide a snippet of the configuration as it would appear in the file? Referencing the IP address might be a bit awkward, but it would be less of a word salad.

Reply:
Rewritten.